### DISCUSSION TITLE
The workbench failed to connect to the server

### DISCUSSION THREAD

**Discussion by Shirtiny** [community/community#18264] [Codespaces] [answered]
![image](https://user-images.githubusercontent.com/49592759/173061235-83963262-aab3-4fb5-96b2-1a7a8957e739.png)
Does anybody know this... ,, I switched to insider version

**Comment by vasilenka** [comment, marked as answer]
Seems like the problem with the latest Insiders build. I'm having problem connecting to codespaces too.
The solution for now is reverting back to previous version of Insiders build, or using the stable version of VS Code.
